packages/shared/lib/clients/orchestrator.ts

```diff
@@ -171,6 +172,8 @@ export class Orchestrator {
                     const errorMsg = `Execute: Failed to parse input '${JSON.stringify(input)}': ${stringifyError(e)}`;
                     const error = new NangoError('action_failure', { error: errorMsg });
                     span.setTag('error', e);
+
+                    metrics.increment(metrics.Types.ACTION_FAILURE);
```

Comment:
we already have the `execute.action` trace above (and the associated metrics) not sure we need to add another metric. Same for webhook.

Reply:
Traces metric does not give the break down per dimensions, it would only give us a volume of actions which is great but not enough in that case
see: https://us3.datadoghq.com/metric/summary?filter=trace.e&metric=trace.execute.action

Reply:
I am not sure I understand what you mean by dimensions. We seems to only be interested in success vs failure which is what `execute.action.hits` and `execute.action.errors` will give you. Am I missing something?

Reply:
hits gives the full volume and errors only gives the error if we tagged the span appropriately (if relevant)
there is no way to get only success (hits - errors)
cf https://us3.datadoghq.com/metric/explorer?fromUser=false&start=1719174058630&end=1719260458630&paused=false#N4Ig7glgJg5gpgFxALlAGwIYE8D2BXJVEADxQEYAaELcqyKBAC1pEbghkcLIF8qo4AMwgA7CAgg4RKUAiwAHOChASAtnADOcAE4RNIKtrgBHPJoQaUAbVBGN8qVoD6gnNtUZCKiOq279VKY6epbINiAiGOrKQdpYZAYgUJ4YThr42gDGSsgg6gi6mZaBZnHKGniqyAUY2QB0cMRwmQRwdbUSUnWM4hrAAFQ87RpOmfgiCAAUAJQgfKCR0bmxWABMickIqel4WTl5iIXFICvlldXatW2Nza3tmZ0iDdrabn2Dw6PjU7M8ALpUVzuPCYULhIGqEEYGKleJzCigCFQmE6NZzAEgDRyNA5UDyDA4hAIfZQHAwUagjQQTKJNCiOBOOSKco4OlQWn0pz0JjKERuDxoeFJCD2TBYRkKfYAIww2ks-x4fEx8jpCAAwlJhDAUCIQWgeEA

Reply:
a simple formula in datadog (hits - errors) will give your the success https://us3.datadoghq.com/metric/explorer?fromUser=false&graph_layout=multi&start=1719174606022&end=1719261006022&paused=false#N4Ig7glgJg5gpgFxALlAGwIYE8D2BXJVEADxQEYAaELcqyKBAC1pEbghkcLIF8qo4AMwgA7CAgg4RKUAiwAHOChASAtnADOcAE4RNIKtrgBHPJoQaUAbVBGN8qVoD6gnNtUZCKiOq279VKY6epbINiAiGOrKQdpYZAYgUJ4YThr42gDGSsgg6gi6mZaBZnHKGniqyAUY2QB0cMRwmQRwdbUSUnWM4hrAAFQ87RpOmfgiCAAUAJQgfKCR0bmxWABMickIqel4WTl5iIXFICvlldXatW2Nza3tmZ0iDdrabn2Dw6PjU7M8ALpUVzuPCYULhDRyNA5UDyDBQhAIfYwIxwaRUWHwxFOUQCUjIAAsfBAQNUIIwMVK8TmFFAEKwUJkIAxiERyjAGHciWZCLg2JEuJQhMBblJmApOjW1NAJLJ4riZAABABaBUrdb-Kh0hlEbms3JQHAwUagjQQTKJNCiXlyRTlHCWqAWq1OehMZQiEVw6lJCD2TBYJw2-YAIw5ln+PCJfvEAGEpMIYCgRCC0DwgA

Reply:
Not very convenient tbh
If you don't mind and to not block this PR I added traces for each entry points in handler + the original metrics, let's compare and see if there is no drawbacks using one or the other 73ad6b95587e5cc25c8c774f7361f07c13589daf